exercise_perseus_render/assets/src/vue/attemptprogress.vue

```diff
@@ -9,9 +9,9 @@
       </div>
     </div>
 
-    <p id="message" v-if="passRatioN === passNum">{{ $tr("get") }} <b>{{passRatioN}} {{ $tr("outof") }} {{passRatioM}}</b> {{ $tr("correct") }}</p>
-    <p id="message" v-if="passNum > 0 && passRatioN !== passNum">{{ $tr("get") }} <b>{{ passNum }}</b> {{ $tr("more") }}</p>
-    <p id="message" v-if="passNum <= 0 && passRatioN !== passNum">{{ $tr("hooray") }}</p>
+    <p id="message" v-if="passRatioM === passNum">{{ $tr("get") }} <b>{{passRatioM}} {{ $tr("outof") }} {{passRatioN}}</b> {{ $tr("correct") }}</p>
+    <p id="message" v-if="passNum > 0 && passRatioM !== passNum">{{ $tr("get") }} <b>{{ passNum }}</b> {{ $tr("more") }}</p>
+    <p id="message" v-if="passNum <= 0 && passRatioM !== passNum">{{ $tr("hooray") }}</p>
```

Comment:
yeah, it will just be some technical debt for now. we can remove the `$trs:` object and change the template to:

```html
    <p id="message" v-if="passRatioM === passNum">Get <b>{{passRatioM}} out of {{passRatioN}}</b> correct!</p>
    <p id="message" v-if="passNum > 0 && passRatioM !== passNum">Get <b>{{ passNum }}</b> more correct!</p>
    <p id="message" v-if="passNum <= 0 && passRatioM !== passNum">Hooray! Good job.</p>```

I did something similar in the import/export wizard because that needed number localization and interpolation also. I'd prefer to do it the right way once we've figured out how to use the syntax.